lib/webhook_system/base_event.rb

```diff
@@ -31,10 +31,11 @@ def as_json
       result = {
         'event_name' => event_name,
         'event_id' => event_id,
+        'data' => {},
       }
       each_attribute do |attribute_name, attribute_method|
         validate_attribute_name attribute_name
-        result[attribute_name.to_s] = public_send(attribute_method).as_json
+        result['data'][attribute_name.to_s] = public_send(attribute_method).as_json
```

Comment:
this is a breaking changes that does not bring much value, we could add attributes to both the current top level and the data sub-section